data/openstudio_standards_data/database_files/level_3_lighting_90_1_2004.json

```diff
@@ -0,0 +1,1766 @@
+[
+    {
+        "method": "BA",
+        "lighting_primary_space_type": "Automotive Facility",
+        "lighting_secondary_space_type": "General",
+        "lighting_power_density": 0.9,
+        "unit": "W/ft2",
```

Comment:
per our conversation, change "unit" to "lighting_power_density_unit", and similar object_name_unit for other variables.

Reply:
addressed